Dockerfile

```diff
@@ -2,17 +2,16 @@ ARG PYTHON_VERSION=3.7
 
 # build typed-ast in separate stage because it requires gcc and libc-dev
 FROM python:${PYTHON_VERSION}-alpine
-COPY constraints.txt .
+COPY requirements.txt constraints.txt ./
 RUN apk add --no-cache gcc libc-dev && \
-    grep typed-ast constraints.txt | pip install -r /dev/stdin
+    pip install -r requirements.txt
```

Comment:
building all of the requirements in a separate stage improves image size and is less prone to breakage